CONTRIBUTING.md

```diff
@@ -3,6 +3,6 @@
 Thanks for you interest onto Sonata projects!
 
 Want to contribute? Please read the following documentation first:
-[dev-kit/doc/CONTRIBUTING.md](https://github.com/sonata-project/dev-kit/blob/master/doc/CONTRIBUTING.md).
+[dev-kit/project/CONTRIBUTING.md](https://github.com/sonata-project/dev-kit/blob/master/project/CONTRIBUTING.md).
```

Comment:
I think this could be even simpler by removing `https://github.com/sonata-project/dev-kit/blob/master/` part.

Could you please update it? Thanks.